tsconfig.json

```diff
@@ -46,9 +46,10 @@
     // "typeRoots": [],                       /* List of folders to include type definitions from. */
     // "types": [],                           /* Type declaration files to be included in compilation. */
     // "allowSyntheticDefaultImports": true,  /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */
-    "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
+    "esModuleInterop": true,                  /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
     // "preserveSymlinks": true,              /* Do not resolve the real path of symlinks. */
     // "allowUmdGlobalAccess": true,          /* Allow accessing UMD globals from modules. */
+    "useUnknownInCatchVariables": false
```

Comment:
I was in that camp... until I got bit by some rogue library throwing a trash value. I won't say that I'm always writing code that adheres to this strict setting but I feel like it's a better choice to have it on. It's also super easy to opt out of it on the call site with:
```ts
} catch (err: any) {
```

and at that point, it's at least in plain sight that we are a little bit on the safe path which is an added bonus for me. This doesn't allow us to do this:
```ts
} catch (err: Error) {
```

but that's also a good thing for me - the cast becomes explicit if we want to reach for it.

And especially with the recent changes in TS 4.9 related to the `in` narrowing the `unknown` type this isn't that bad.

```suggestion
```

Not a hill I'm willing to die on but this one doesn't annoy me enough to turn it off.

Reply:
Fair enough! I'll remove this from PR